src/content/en/2021/css.md

```diff
@@ -2,7 +2,7 @@
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: CSS
 description: TODO
-authors: []
+authors: [meyerweb]
 reviewers: []
 analysts: []
```

Comment:
@meyerweb can you suggest edits to the Authors, Reviewers, and Analysts properties so that they contain everyone who contributed to the CSS chapter, roughly ordered by those with the most significant contributions first? This is what readers will see at the top of the chapter so we want to be sure to give appropriate credit to everyone who helped.